README.md

```diff
@@ -77,16 +86,13 @@ Processes an input string using [Markdown](https://markdown-it.github.io).
 
 Output: `<p>A paragraph with some <em>emphasis</em>.</p>\n`
 
-#### slug
+#### slug (deprecated)
```

Comment:
Often with deprecations there is time to switch off. I'm testing this with trivet and getting a bunch of errors because it uses slug-filter in the eleventy.js file. 

- One option would be to mention here in the docs that the slug-filter file has been removed. 
- Another options would be to keep all those but make an internal call to slugify, essentially just passing along the data to it. Instead of throwing an error you'd log a warning about slug being deprecated and etc. We'd then remove it in the next major version.

Reply:
@jobara are you testing with a new install of Trivet? That error will be fixed once this PR is merged. My preference would be the first option, noting that `slug-filter.js` has been removed.

Reply:
> @jobara are you testing with a new install of Trivet? That error will be fixed once [this PR](https://github.com/fluid-project/trivet/pull/235) is merged. My preference would be the first option, noting that `slug-filter.js` has been removed.

I'm just testing with trivet itself, and npm linking the plugin.

Reply:
Can you check out this branch from my fork and try it? https://github.com/greatislander/trivet/tree/fix/slug-regex